packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -755,6 +793,18 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
         });
     }
 
+    private handleThemeUpdate = (e: any): void => {
+        if (this.props.onUpdateTheme) {
+            this.props.onUpdateTheme(e.target.value);
+        }
+
+        const found: IThemeValues = this.props.themes.find(function(item: IThemeValues): boolean { return item.name === e.target.value; });
```

Comment:
I would rename this, found is not very intuitive for what it is, `selectedTheme` might be better